encord/orm/label_row.py

```diff
@@ -393,6 +396,7 @@ class LabelRowMetadataDTO(BaseDTO):
     data_title: str
     data_type: str
     data_link: Optional[str] = None
+    backing_item_uuid: str
```

Comment:
@Jim-Encord, I don't have full context on this, but should it be optional? It seems to be optional in other places

Reply:
yeah good spot. I originally was hoping that it could be a required arg throughout but then that breaks the previous format so should probably be optional throughout which is sad.